app/helpers/application_helper.rb

```diff
@@ -37,7 +37,9 @@ def hot_javascript_tag(filename)
   # :nocov:
 
   def hot_javascript_path(filename)
-    return "http://localhost:8080/#{filename}.js" if Features.hot_loading?
```

Comment:
Would this work as `/assets/javascripts/#{filename}.js` (which would delegate the domain to the proxy middleware)?

Reply:
ah, I think I misunderstood you. Did you mean to replace this with `/assets/javascripts/#{filename}.js`?

Reply:
Yeah, that that's what I meant.